provider-ci/internal/pkg/templates/bridged-provider/Makefile

```
@@ -119,9 +119,11 @@ install_plugins: .pulumi/bin/pulumi
 lint_provider: provider
 	cd provider && golangci-lint run -c ../.golangci.yml
 
-provider: tfgen install_plugins
+provider_no_deps:
 	(cd provider && go build $(PULUMI_PROVIDER_BUILD_PARALLELISM) -o $(WORKING_DIR)/bin/$(PROVIDER) -ldflags "-X $(PROJECT)/$(VERSION_PATH)=$(VERSION)#{{if .Config.providerVersion}}# -X #{{ .Config.providerVersion }}#=$(VERSION)#{{end}}#" $(PROJECT)/$(PROVIDER_PATH)/cmd/$(PROVIDER))
 
+provider: tfgen install_plugins provider_no_deps
```

The only reason that `provider` depends on `install_plugins` AFAIK is that `tfgen` depends on `install_plugins`. Since we already have `tfgen: install_plugins`, I think we can drop this dependency.
```suggestion
provider: tfgen provider_no_deps
```